### PR TITLE
[Rated Disabilities] - Migrate v1 components to v3 components 

### PR DESCRIPTION
**Pull request by pmclaren19**
## Summary

Updated the following files to no longer have `uswds='false'`
- src/applications/rated-disabilities/components/FeatureFlagsLoaded.jsx
- src/applications/rated-disabilities/components/Learn.jsx
- src/applications/rated-disabilities/components/MVIError.jsx
- src/applications/rated-disabilities/components/NeedHelp.jsx
- src/applications/rated-disabilities/components/RatedDisabilityList.jsx
- src/applications/rated-disabilities/components/SortSelect.jsx
- src/applications/rated-disabilities/components/TotalRatedDisabilities.jsx
- src/applications/rated-disabilities/components/TotalRatingStates.jsx

Tested the following files that have V3 components with `uswds='true'` defaulted
- src/applications/rated-disabilities/components/MVIError.jsx
- src/applications/rated-disabilities/components/RatedDisabilityList.jsx
- src/applications/rated-disabilities/components/SortSelect.jsx
- src/applications/rated-disabilities/components/TotalRatingStates.jsx

Updated the following files that already have `uswds='true'` to no longer have that and use the default true
- src/applications/rated-disabilities/components/NoCombinedRating.jsx
- src/applications/rated-disabilities/components/RatingLists/NoRatings.jsx
- src/applications/rated-disabilities/components/ServerError.jsx

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#76124

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| MVIError.jsx - warning| Screenshot 2024-02-23 at 12 49 52 PM | Screenshot 2024-02-23 at 12 52 07 PM |
| RatedDisabilityList.jsx - info | Screenshot 2024-02-26 at 10 14 44 AM | Screenshot 2024-02-26 at 10 15 27 AM |
| RatedDisabilityList.jsx - error | Screenshot 2024-02-26 at 10 19 42 AM | Screenshot 2024-02-26 at 10 19 19 AM |
| SortSelect.jsx | https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/181f9a5a-f23b-435e-8107-2420c08b6415 | https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/0f796f64-aa9f-46ff-902a-9a10cf1e46a9 |
| TotalRatingStates.jsx - info | Screenshot 2024-02-26 at 11 55 53 AM | Screenshot 2024-02-26 at 11 56 20 AM |
| TotalRatingStates.jsx - va-summary-box | Screenshot 2024-02-26 at 11 57 50 AM | Screenshot 2024-02-26 at 12 26 02 PM |
| TotalRatingStates.jsx - error | Screenshot 2024-02-26 at 12 37 47 PM | Screenshot 2024-02-26 at 12 38 32 PM |

## What areas of the site does it impact?

Rated Disabilities

## Acceptance criteria

- [x] All V1 components with a V3 version are migrated to that version of the component.
- [x] All V1 components that do not have a V3 version do not have a prop set to false
- [x] Post migration, the UI meets the BMT team's design and Ux standards.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

Steps to repo MVIError.jsx - default =>  I had to override the code to get this one to display
Steps to repo RatedDisabilityList.jsx - default => go to user vets.gov.user+0@gmail.com
Steps to repo RatedDisabilityList.jsx - error => I had to override the code to get this one to display
Steps to repo SortSelect.jsx => ratedDisabilitiesSortAbTest feature flag is enabled
Steps to repo TotalRatingStates.jsx - info  => go to user vets.gov.user+0@gmail.com
Steps to repo TotalRatingStates.jsx - va-summary-box  => go to user vets.gov.user+1@gmail.com
Steps to repo TotalRatingStates.jsx - error  => I had to override the code to get this one to display

### CODE REVIEW

**Comment by pmclaren19** [comment]
Added code changes. Blocked on merging until platform team merges their pr on 3/4 @HeatherWidmont @jacobworrell

**Comment by skylerschain** [comment]
Images LGTM 👍

**Review by samcoforma — DISMISSED** [review]

**Review by micahchiang — DISMISSED** [review]

File: src/applications/rated-disabilities/components/NeedHelp.jsx
Comment (samcoforma) [inline src/applications/rated-disabilities/components/NeedHelp.jsx:10]: Should this say `If you have` instead of `If have`?

**Review by micahchiang — DISMISSED** [review]

**Review by samcoforma — CHANGES_REQUESTED** [review]

File: src/applications/rated-disabilities/components/NeedHelp.jsx
Comment (pmclaren19) [reply src/applications/rated-disabilities/components/NeedHelp.jsx]: updated !

**Review by samcoforma — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Pull request merged**
